docker/workspace-build/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # Assumptions:
 # - ROS_DISTRO is defined with the name of a distribution installed under /opt/ros/
 #
-ARG ROS_VERSION
+ARG ROS_VERSION="foxy"
```

Comment:
Help me recall: what are the reasons we cannot upgrade from foxy? mmdet was one; were there others?

Reply:
I was wondering if we should revisit this

Reply:
mmdet/mmpose specifically.

Reply:
Right. Noted, thanks.